tidb-server/server/conn.go

```diff
@@ -86,6 +86,7 @@ func (cc *clientConn) handshake() error {
 		cc.writeError(err)
 		return errors.Trace(err)
 	}
+	log.Info("[conn] capability %v", cc.capability)
```

Comment:
I am not sure how to write unit test for connection. 
I change the title to DNM and will update this PR latter.